MGSpotyView/MGViewControllerDataSource.m

```diff
@@ -0,0 +1,56 @@
+//
+//  MGViewControllerDataSource.m
+//  MGSpotyView
+//
+//  Created by Daniele Bogo on 08/08/2015.
+//  Copyright (c) 2015 Matteo Gobbi. All rights reserved.
+//
+
+#import "MGViewControllerDataSource.h"
+#import "MGSpotyViewController.h"
+
+
+@implementation MGViewControllerDataSource
+
+
+#pragma mark - MGSpotyViewControllerDataSource
+
+- (NSInteger)numberOfSectionsInSpotyViewController:(MGSpotyViewController *)spotyViewController
+{
+    NSInteger mySections = 1;
+    return mySections + 1;
+}
+
+- (NSInteger)spotyViewController:(MGSpotyViewController *)spotyViewController
+           numberOfRowsInSection:(NSInteger)section
+{
+    return (section == 1) ? 20 : 0;
```

Comment:
idem, here the user could do `if (section == 0)` but for us, is the section 1, as section 0 is protected.